docker/php/Dockerfile

```diff
@@ -67,16 +73,9 @@ RUN extBuildDeps=" \
     echo 'ForwardAgent yes' >> /etc/ssh/ssh_config  && \
     curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer && \
     composer clear-cache && \
-    mkdir -p /root/.ssh/ && \
-    ssh-keyscan github.com >> /root/.ssh/known_hosts
+    mkdir -p /root/.ssh/
 
-RUN version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") \
-    && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$version \
-    && mkdir -p /tmp/blackfire \
-    && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire \
-    && mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get('extension_dir');")/blackfire.so \
-    && printf "extension=blackfire.so\nblackfire.agent_socket=tcp://blackfire:8707\n" > $PHP_INI_DIR/conf.d/blackfire.ini \
-    && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz
+COPY know_hosts /root/.ssh/known_hosts
```

Comment:
Let's remove this, known hosts are not our responsibility.

Reply:
Ho ! Yes !!!!